proof-general.el

```diff
@@ -16,7 +16,7 @@
 
 ;; This software is free software; you can redistribute it and/or
 ;; modify it under the terms of the GNU General Public License
-;; version 2, as published by the Free Software Foundation.
+;; version 3, as published by the Free Software Foundation.
```

Comment:
I skimmed the PR diff and it seems the only missing words are these ones:

```suggestion
;; version 3 or later, as published by the Free Software Foundation.
```

But maybe you could just rephrase a bit the whole paragraph to have the very same text as other files?

```
;; This is free software; you can redistribute it and/or modify
;; it under the terms of the GNU General Public License as published by
;; the Free Software Foundation; either version 3, or (at your option)
;; any later version.
```